src/Selenium2Driver.php

```diff
@@ -68,25 +70,31 @@ class Selenium2Driver extends CoreDriver
     /**
      * Instantiates the driver.
      *
-     * @param string $browserName         Browser name
-     * @param array  $desiredCapabilities The desired capabilities
-     * @param string $wdHost              The WebDriver host
+     * @param string $browserName Browser name
+     * @param array $desiredCapabilities The desired capabilities
+     * @param string $wdHost The WebDriver host
      */
     public function __construct($browserName = 'firefox', $desiredCapabilities = null, $wdHost = 'http://localhost:4444/wd/hub')
     {
         $this->setBrowserName($browserName);
         $this->setDesiredCapabilities($desiredCapabilities);
-        $this->setWebDriver(new WebDriver($wdHost));
+        $this->wdHost = $wdHost;
         $this->xpathEscaper = new Escaper();
     }
 
     /**
      * Sets the browser name
      *
      * @param string $browserName the name of the browser to start, default is 'firefox'
+     *
+     * @throws DriverException
      */
     protected function setBrowserName($browserName = 'firefox')
     {
+        if ($this->started) {
```

Comment:
Fact, that no exception was thrown before proves that nobody really does that.

If you'd like, then you can move exception throwing into separate PR since it's not related to switching to Facebook's WebDriver in any way.